src/Symfony/Bundle/FrameworkBundle/Logger/Logger.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Logger;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LogLevel;
+
+/**
+ * Minimalist PSR-3 logger designed to write in stdout, stderr or any other stream.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
```

Comment:
Could be marked as @internal, right?

Reply:
Indeed. It may be useful in other contexts than in FrameworkBundle, but this class is so simple that a copy/paste should be good enough.